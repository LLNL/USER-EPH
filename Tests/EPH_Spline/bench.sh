#!/bin/bash

for i in {1..10} ; do ./test ; done | grep Elapsed | gawk 'BEGIN {a=0.0;n=0} {a=a+$3; n++; print $3, a, n} END {a=a/n; print "Average ", a}'

