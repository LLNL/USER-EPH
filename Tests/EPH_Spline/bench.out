O2 version
6.361 6.361 1
6.33 12.691 2
6.261 18.952 3
6.236 25.188 4
6.793 31.981 5
6.586 38.567 6
6.244 44.811 7
6.777 51.588 8
6.849 58.437 9
6.243 64.68 10
Average  6.468
O3 version
6.712 6.712 1
7.063 13.775 2
6.24 20.015 3
6.277 26.292 4
6.234 32.526 5
6.254 38.78 6
6.303 45.083 7
6.264 51.347 8
6.25 57.597 9
6.242 63.839 10
Average  6.3839
Ofast version
5.764 5.764 1
5.493 11.257 2
5.837 17.094 3
5.528 22.622 4
5.464 28.086 5
5.454 33.54 6
5.453 38.993 7
5.458 44.451 8
5.459 49.91 9
5.452 55.362 10
Average  5.5362
ndebug version
5.45 5.45 1
5.529 10.979 2
5.652 16.631 3
5.457 22.088 4
5.478 27.566 5
5.439 33.005 6
5.464 38.469 7
5.489 43.958 8
5.628 49.586 9
5.467 55.053 10
Average  5.5053
unsafe version
5.207 5.207 1
5.204 10.411 2
5.222 15.633 3
5.363 20.996 4
5.192 26.188 5
5.192 31.38 6
5.205 36.585 7
5.248 41.833 8
5.174 47.007 9
5.165 52.172 10
Average  5.2172
